### ISSUE TITLE
BAD_TERM on exog_action with multiple effects

### ISSUE THREAD

**Issue by Setcab** [MASKOR/gologpp#26]
```
Enabling ECLiPSe debugging.
Socket created at address agamemnon.kbsg.rwth-aachen.de / 45487
13:59:50.775351 gologpp_agent: ... initialization done
exog_action(incoming_orders(_523034, _523032, _523035, _523033, _523036, _523031, _523030))
poss(incoming_orders(_523282, _523280, _523283, _523281, _523284, _523279, _523278), true)
causes_val(incoming_orders(_523531, _523529, _523532, _523530, _523533, _523528, _523527), order_complexity(_523531), _523529, true)
causes_val(incoming_orders(_523531, _523529, _523532, _523530, _523533, _523528, _523527), order_base_color(_523531), _523532, true)
causes_val(incoming_orders(_523531, _523529, _523532, _523530, _523533, _523528, _523527), order_ring_colors(_523531), _523530, true)
causes_val(incoming_orders(_51, _49, _52, _50, _53, _48, _47), order_cap_color(_51), _53, true)
type error in recordz_body('REC'(16'5c0e2040), causes_val(incoming_orders([_51|_52], BAD_TERM_0x7fad54000310_0x60001feb551ff424, BAD_TERM_0x7fad54000340_0x60001feb551ff430, BAD_TERM_0x7fad54000320_0x60001feb551ff428, BAD_TERM_0x7fad54000350_0x60001feb551ff530, BAD_TERM_0x7fad54000300_0x60001feb551ff420, BAD_TERM_0x7fad540002f0_0x60001feb551ff41c), order_begin([_51|_52]), BAD_TERM_0x7fad54000300_0x60001feb551ff420, true) :- true, sepia_kernel)
ERROR: In compiling term:
  Query exited (abort): ?- assertz(causes_val(incoming_orders([_51|_52], BAD_TERM_0x7fad54000310_0x60001feb551ff424, BAD_TERM_0x7fad54000340_0x60001feb551ff430, BAD_TERM_0x7fad54000320_0x60001feb551ff428, BAD_TERM_0x7fad54000350_0x60001feb551ff530, BAD_TERM_0x7fad54000300_0x60001feb551ff420, BAD_TERM_0x7fad540002f0_0x60001feb551ff41c), order_begin([_51|_52]), BAD_TERM_0x7fad54000300_0x60001feb551ff420, true))
13:59:50.780469 gologpp_agent: Eclipse Error: Failed to compile_term/1
13:59:50.782442 FawkesMainThread: Fawkes 1.3.0 startup complete
```

**Comment by Setcab** [comment]
Code of corresponding action:
https://github.com/carologistics/fawkes-robotino-labhlap2019/blob/labhlap3/old_syntax/src/gologpp/rcll/rcll.gpp#L174-L200

**Comment by vmatare** [comment]
bad term cleanly formatted:
```prolog
causes_val(
	incoming_orders(
		[_51|_52]
		, BAD_TERM_0x7fad54000310_0x60001feb551ff424
		, BAD_TERM_0x7fad54000340_0x60001feb551ff430
		, BAD_TERM_0x7fad54000320_0x60001feb551ff428
		, BAD_TERM_0x7fad54000350_0x60001feb551ff530
		, BAD_TERM_0x7fad54000300_0x60001feb551ff420
		, BAD_TERM_0x7fad540002f0_0x60001feb551ff41c
	)
	, order_begin([_51|_52])
	, BAD_TERM_0x7fad54000300_0x60001feb551ff420
	, true
) :- true
```

**Comment by vmatare** [comment]
should be fixed by 20225af56f7ecc29fd53fdbd6c3f590777705602. Waiting to confirm with lab tests.

**Comment by vmatare** [comment]
seems to be finally fixed.
